QuestionAnswering/QA_ICL/frontend/README.md

```diff
@@ -1,4 +1,6 @@
-# Frontend Development for Marie
+# Marie's Frontend and Mock Server
 
-- [`mock_backend`](./mock_backend/): minimal Express server to mock Marie's backend APIs
-- [`next_app_marie`](./next_app_marie/): Next app for Marie's frontend
+This directory contains the following: 
+
+- [`mock_backend`](./mock_backend/): A minimal Express server to mock backend APIs.
+- [`next_app_marie`](./next_app_marie/): A Next.js app fo
```

Comment:
Looks like half of this was accidentally deleted?

Reply:
I have made the change to complete the second bullet point.